lib/select-builder.js

```diff
@@ -1,89 +1,74 @@
 'use strict';
 
-const { iter } = require('@metarhia/common');
-
 const { QueryBuilder } = require('./query-builder');
 
-const allowedConditions = new Set(['=', '<>', '<', '<=', '>', '>=', 'LIKE']);
-
-const supportedOps = {
-  select: Set,
-  innerJoin: Array,
-  selectDistinct: null,
-  count: Array,
-  avg: Array,
-  min: Array,
-  max: Array,
-  sum: Array,
-  where: Array,
-  groupBy: Set,
-  orderBy: Set,
-  from: Set,
-  limit: null,
-  offset: null,
-};
+const compareOperators = ['=', '<>', '<', '<=', '>', '>=', 'LIKE'];
+const setOperators = ['= ANY', 'IN', 'NOT IN'];
+const operators = [...compareOperators, ...setOperators];
 
-const functionHandlers = {
-  count: op => `count(${op.field})`,
-  avg: op => `avg(${op.field})`,
-  min: op => `min(${op.field})`,
-  max: op => `max(${op.field})`,
-  sum: op => `sum(${op.field})`,
-};
+const sqlFunctions = ['count', 'avg', 'min', 'max', 'sum'];
```

Comment:
How will this support functions with custom and multiple arguments (i.e. https://www.postgresql.org/docs/12/functions-string.html or in general https://www.postgresql.org/docs/12/functions.html)?

Reply:
I see, I'd propose different solution